package.json

```
@@ -8,20 +8,20 @@
     "build-report": "cross-env NODE_ENV=production BUILD_REPORT=true webpack --progress",
     "watch": "yarn webpack serve --mode development --env development",
     "start": "node server.js",
-    "dev": "cross-env NODE_ENV=development node server.js",
-    "test": "echo \"Error: no test specified\" && exit 1",
+    "dev": "cross-env NODE_ENV=development nodemon --exec node server.js",
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Missing trailing comma

```suggestion
    "test": "echo \"Error: no test specified\" && exit 1",
```